tools/WorkerHarness/src/WorkerHarness.Core/Worker/IWorkerDescriptionBuilder.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
```

Comment:
Add copyright header to all files. 
https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Binding/BindingContext.cs#L1